### DISCUSSION TITLE
Find and Replace

### DISCUSSION THREAD

**Discussion by sarahf16** [GitbookIO/community#327] [Feature Requests]
### Ask a question, start a conversation, or make an annoucement

For quite a lot of the updates I need to do, I would have used 'Find and Replace' in Word. Now we have moved the documentation over to GitBook this is taking me a lot longer as I need to change each instance individually. Has anyone found a workaround to do this in GitBook or GitLab, as we sync the content to the master copy here?

I can see this has already been logged as a feature request - is this something that is likely to be implemented?

Thanks

Sarah

**Comment by aikoii** [comment]
Hey @sarahf16! We want to  have this at some point 🙂.

If you're on GitHub and using Git Sync, you can update one file or repo at a time. To make changes, download the repo and open it in a text editor like VS Code, etc. Use `Find and Replace` to update the text, save the changes, commit them in Git and push to the remote Repo to sync back to GitBook.

⚠️ Important thing to note is that the word you're replacing may also appear in config files, like `SUMMARY.md` or `.gitbook.yaml`, which could cause broken links.

**Reply by sarahf16** [reply]
Thank you for this, I use GitLab, so will have a look to see if they have something similar to GitHub.
